NitroxClient/GameLogic/FMOD/FMODSystem.cs

```diff
@@ -11,19 +12,16 @@ namespace NitroxClient.GameLogic.FMOD
 {
     public class FMODSystem
     {
-        private readonly Dictionary<string, SoundData> assetWhitelist = new Dictionary<string, SoundData>();
-
+        private readonly Dictionary<string, SoundData> assetWhitelist = new();
         private readonly IPacketSender packetSender;
 
         public FMODSystem(IPacketSender packetSender)
         {
             this.packetSender = packetSender;
-
-            string soundsWhitelist = Properties.Resources.soundsWhitelist;
-
+            string soundsWhitelist = Resources.soundsWhitelist;
             if (string.IsNullOrWhiteSpace(soundsWhitelist))
             {
-                Log.Error(new NullReferenceException(), "[FMODSystem]: soundsWhitelist.csv is null or whitespace");
+                Log.Error(new Exception(), "[FMODSystem]: soundsWhitelist.csv is null or whitespace");
```

Comment:
Why changing it to a generic exception? If NRE isn't fitting I would remove it completely and only log the message.

Reply:
Because it's null or whitespace, not just null. But still wanted the stack trace.

Reply:
The stack trace is useless, with this `new` on same line. Will remove it.